src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -788,7 +792,25 @@ public string OutputResultsCacheFile
         /// <summary>
         /// Gets or sets a value indicating whether the build process should run as low priority.
         /// </summary>
-        public bool LowPriority { get; set; }
+        public bool LowPriority
```

Comment:
What issue is this solving? Won't this cause VS to trip into low-priority mode if it triggers a build in low-pri?

Reply:
You're right. Maybe the right thing here is to disconnect from child processes when switching into low-pri, forcing VS to find new nodes to connect to or make more?